src/Provider/IslandoraServiceProvider.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Islandora\Crayfish\Commons\Provider;
+
+use Islandora\Crayfish\Commons\CmdExecuteService;
+use Islandora\Crayfish\Commons\FedoraResourceConverter;
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Silex\Provider\DoctrineServiceProvider;
+use Silex\Provider\MonologServiceProvider;
+use Silex\Provider\ServiceControllerServiceProvider;
+use Silex\Provider\SecurityServiceProvider;
+use Islandora\Chullo\FedoraApi;
+use Islandora\Crayfish\Commons\Syn\SettingsParser;
+use Islandora\Crayfish\Commons\Syn\JwtAuthenticator;
+use Islandora\Crayfish\Commons\Syn\JwtFactory;
+
+class IslandoraServiceProvider implements ServiceProviderInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function register(Container $container)
+    {
+        // Register services we rely on
+        $container->register(new MonologServiceProvider());
+        $container->register(new ServiceControllerServiceProvider());
+        $container->register(new SecurityServiceProvider());
+        $container->register(new DoctrineServiceProvider());
+
+        // Configure external services
+        $container['monolog.logfile'] = function ($container) {
+            return strtolower($container['crayfish.log.level']) == 'none' ? null : $container['crayfish.log.file'];
+        };
+        $container['monolog.level'] = function ($container) {
+            return $container['crayfish.log.level'];
```

Comment:
We don't have a "crayfish" application that runs, so I think this should probably be `$container['gemini.log.level']` or `$container['houdini.log.level']` or perhaps I misunderstand the YamlConfigServiceProvider. But I am getting..
```
[Tue Apr 18 02:57:48 2017] PHP Fatal error:  Uncaught InvalidArgumentException: Identifier "crayfish.log.level" is not defined. in /home/ubuntu/all_claw/Crayfish/Gemini/vendor/pimple/pimple/src/Pimple/Container.php:96
Stack trace:
#0 /home/ubuntu/all_claw/Crayfish/Gemini/vendor/islandora/crayfish-commons/src/Provider/IslandoraServiceProvider.php(33): Pimple\Container->offsetGet('crayfish.log.le...')
#1 /home/ubuntu/all_claw/Crayfish/Gemini/vendor/pimple/pimple/src/Pimple/Container.php(113): Islandora\Crayfish\Commons\Provider\IslandoraServiceProvider->Islandora\Crayfish\Commons\Provider\{closure}(Object(Silex\Application))
#2 /home/ubuntu/all_claw/Crayfish/Gemini/vendor/silex/silex/src/Silex/Provider/MonologServiceProvider.php(94): Pimple\Container->offsetGet('monolog.logfile')
#3 /home/ubuntu/all_claw/Crayfish/Gemini/vendor/pimple/pimple/src/Pimple/Container.php(113): Silex\Provider\MonologServiceProvider->Silex\Provider\{closure}(Object(Silex\Application))
#4 /home/ubuntu/all_claw/Crayfish/Gemini/vendor/silex/silex/src/Silex/Provider/MonologServicePro in /home/ubuntu/all_claw/Crayfish/Gemini/vendor/pimple/pimple/src/Pimple/Container.php on line 96
```